src/HtmlUtils.js

```diff
@@ -136,6 +138,37 @@ var sanitizeHtmlParams = {
             attribs.rel = 'noopener'; // https://mathiasbynens.github.io/rel-noopener/
             return { tagName: tagName, attribs : attribs };
         },
+        '*': function(tagName, attribs) {
+            // Delete any style previously assigned
+            delete attribs.style;
```

Comment:
Transforming is done before attributes are stripped (sensibly). I shall comment.